app/Presenters/Admin/ExhibitionPresenter.php

```diff
@@ -67,6 +68,21 @@ public function headerType()
         }
     }
 
+    public function position()
+    {
+        $page = Page::forType('Exhibitions and Events')->with('apiElements')->first();
+
+        $featurePos = $page->exhibitionsExhibitions->search(function ($item) {
+            return $item->datahub_id == $this->entity->datahub_id;
+        });
+
+        $listingPos = $page->exhibitionsCurrent->search(function ($item) {
+            return $item->datahub_id == $this->entity->datahub_id;
+        });
+
+        return $featurePos !== false ? $featurePos : ($listingPos !== false ? $listingPos + 2 : -1);
```

Comment:
Can you leave a comment explaining the "2" and "-1" in `($listingPos !== false ? $listingPos + 2 : -1)`

Reply:
Yes, that is confusing. I added a comment to clarify:

```
// $featurePos represents the first two spots on the exhibitions landing page
// $listingPos represents the remaining exhibitions listing in the landing page
// If the exhibition is the featured position, return its value
// If the exhibition is listed below the featured two, bump it's index by two and return it
// Otherwise, the exhibition is not listed on the exhibition landing page. Return -1.
```